src/ui/components/shared/SettingsModal/SettingsModal.tsx

```diff
@@ -56,6 +56,14 @@ const settings: Settings = [
         disabled: false,
         needsRefresh: false,
       },
+      {
+        label: "Enable repainting",
+        type: "checkbox",
+        key: "enableRepaint",
+        description: "Repaint the DOM whenever we jump in time",
```

Comment:
I wonder what others think of this description. My concern is that it could make people more confused about the default behavior. Like wait, if this off what are we showing?
 
I think I would prefer something more vague such as "Repaint the DOM on demand" which is harder to confuse with the default behavior.